synapse/util/__init__.py

```diff
@@ -17,13 +17,16 @@
 import re
 
 import attr
+from canonicaljson import json
 
 from twisted.internet import defer, task
 
 from synapse.logging import context
 
 logger = logging.getLogger(__name__)
 
+json_encoder = json.JSONEncoder(separators=(",", ":"))
```

Comment:
please could you give this a docstring? where is it intended that it be used?

Reply:
oh, docstrings don't really work on variables like this. A comment instead?

Reply:
```suggestion
# Create a custom encoder to reduce the whitespace produced by JSON encoding.
json_encoder = json.JSONEncoder(separators=(",", ":"))
```